internal/pkg/template/input/input_test.go

```diff
@@ -21,7 +21,7 @@ func TestTemplateInputsValidateDefinitions(t *testing.T) {
 	}}
 	err := inputs.ValidateDefinitions()
 	assert.Error(t, err)
-	assert.Equal(t, `id can only contain alphanumeric characters, dots and underscores`, err.Error())
+	assert.Equal(t, `[0].id can only contain alphanumeric characters, dots and underscores`, err.Error())
```

Comment:
Tu sa vsade validuju `slice`, ale v msg to nebolo spravne.

Vyledna hlaska pre uzivatela neskor bude napr.
```
invalid inputs in inputs.json:
- [0].id can only contain alphanumeric characters, dots and underscores
- [4].default must be the same type as type or options
```